lib/puppet/pops/loaders.rb

```diff
@@ -195,7 +196,10 @@ def create_environment_loader(environment)
       loader = Loader::SimpleEnvironmentLoader.new(@runtime3_type_loader, loader_name)
     else
       # View the environment as a module to allow loading from it - this module is always called 'environment'
-      loader = Loader::ModuleLoaders.module_loader_from(@runtime3_type_loader, self, 'environment', env_conf.path_to_env)
+      # TODO: temporary hack
+      path_to_env = env_conf.path_to_env
```

Comment:
@hlindberg this is the part that I didn't quite get to figuring out ... the Environment has fully expanded paths at this point, *but* the call to `Puppet.lookup(:environments).get_conf(environment.name).path_to_env` returns an 8.3.

I want to be able to remove the expansion call here.